xAPI-About.md

```diff
@@ -441,21 +441,22 @@ This section explains and shows graphically how different pieces of xAPI can fit
 
 ##### Tracking Experiences
 
+###### Figure 1: Data Flow in xAPI
 ![Data Flow in xAPI](/xAPIDataFlow.jpg)
 
 Figure 1 shows the tracking of learning experiences. A learner has a Learning Experience. This experience could take place in an 
 online course, it could be on the job or it could be part of recreation. It really could be anything. This experience is tracked, 
-on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider may also be responsible for the 
-trusted relationship between the experience and the learner, and perhaps even launching “content” for the learner and digital 
-rights associated with the content, but shouldn't be viewed as boundaries or even "the norm".  
+on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider can also be responsible for the 
```

Comment:
This makes me wonder about something that's probably outside scope, here, but... do we not expect "Learning Record Provider" and "Learning Record Consumer" to become abbreviated as "LRP" and "LRC", respectively?